torchbenchmark/models/pplbench_beanmachine/__init__.py

```diff
@@ -0,0 +1,53 @@
+import time
+import random
+import torch
+
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import OTHER
+from typing import Tuple
+
+from pplbench.ppls.beanmachine import robust_regression
+from pplbench.models.robust_regression import RobustRegression
+from pplbench.ppls.beanmachine.inference import MCMC
+
+class Model(BenchmarkModel):
+    task = OTHER.OTHER_TASKS
+    DEFAULT_EVAL_BSIZE = 1
```

Comment:
If batch size is not adjustable, please add a new member variable `ALLOW_CUSTOMIZE_BSIZE = False` here,
like in https://github.com/pytorch/benchmark/blob/main/torchbenchmark/models/drq/__init__.py#L90

Reply:
Done